scarb/src/bin/scarb/commands/init.rs

```diff
@@ -24,7 +25,12 @@ pub fn run(args: InitArgs, config: &Config) -> Result<()> {
             } else {
                 VersionControl::Git
             },
-            snforge: args.snforge,
+            snforge: matches!(
+                Ok(args.test_runner)
+                    .transpose()
+                    .unwrap_or_else(ask_for_test_runner)?,
```

Comment:
this sounds invalid. we cannot ask in non-interactive shell. is your library handling this under the hood or so?

Reply:
We can check if it is a `TTY` device via the [`atty`](https://crates.io/crates/atty) crate. If it is not, we can either go with the default option or exit with an error. What do you think?

Reply:
`atty` has a security hole and in general is long-time abandoned in the ecosystem

https://doc.rust-lang.org/stable/std/io/trait.IsTerminal.html is the way to go :)